ras/sosreport.py

```diff
@@ -75,13 +74,6 @@ def test_short(self):
         self.run_cmd("sosreport -h", False)
         self.run_cmd("sosreport -l", False)
         self.run_cmd("sosreport --batch --tmp-dir=%s --verify" % directory_name)
-
-        ticket_id = self.params.get('ticket_id', default='testid')
```

Comment:
Can you also remove the same from the yaml file?

Reply:
sure